app/src/pages/inside/common/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -231,6 +240,17 @@ export class Hamburger extends Component {
     });
   };
 
+  changeImportantState = (markAsImportant) => {
+    this.props.showModal({
+      id: markAsImportant ? 'markAsImportantModal' : 'unmarkAsImportantModal',
+      data: {
+        activeProject: this.props.projectId,
+        launch: this.props.launch,
+        updateLaunchLocally: (data) => this.props.updateLaunchLocallyAction(data),
```

Comment:
From the modal and its internal hook perspective it's just a callback.
`updateLaunchLocallyAction` action can be replaced with smth else that will force the renaming of all it's usages inside of the modals and hook.
So I suggest to rename this callback to just `onSubmit` or `onSuccess` to be more flexible.